lib/bonuses/HeroBonus.h

```diff
@@ -9,10 +9,8 @@
  */
 #pragma once
 
-#include "GameConstants.h"
-#include "JsonNode.h"
-#include "battle/BattleHex.h"
-#include <limits>
+#include "../GameConstants.h"
+#include "../JsonNode.h"

```

Comment:
Only needed for BonusParams struct
Suggestion: move BonusParams to a separate file, remove include

Reply:
done